website/content/docs/k8s/installation/install.mdx

```diff
@@ -15,58 +15,54 @@ For pure-Kubernetes workloads, this enables Consul to also exist purely
 within Kubernetes. For heterogeneous workloads, Consul agents can join
 a server running inside or outside of Kubernetes.
 
-This page starts with a large how-to section for various specific tasks.
-To learn more about the general architecture of Consul on Kubernetes, scroll
-down to the [architecture](/docs/k8s/installation/install#architecture) section.
-If you would like to get hands-on experience testing Consul as a service mesh
-for Kubernetes, check the guides in the [Getting Started with Consul service
-mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) track.
+You can install Consul on Kubernetes using the Helm chart or by using the Consul K8s CLI tool <sup>ALPHA</sup>.
+Refer to the [architecture](/docs/k8s/installation/install#architecture) section to learn more about the general architecture of Consul on Kubernetes.
+For a hands-on experience with Consul as a service mesh
+for Kubernetes, follow the [Getting Started with Consul service
+mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
```

Comment:
Could we perhaps include a TOC so one could quickly click on Helm Chart installation or Consul K8s CLI Installation?

Reply:
The docs already have a TOC built into the UI
<img width="603" alt="2021-09-16_12-49-07" src="https://user-images.githubusercontent.com/47586768/133676636-0937c936-79ec-48d3-8d30-88f41ae2d246.png">

You think we should hard code one into the page? I believe the plan is to make the built-in TOCs more noticeable, but I don't know how far away we are from that design change.

Reply:
> Could we perhaps include a TOC so one could quickly click on Helm Chart installation or Consul K8s CLI Installation?

Was thinking of something like this instead. I had nod ideas there was a TOC so it was hard for a normal user to navigate quickly. 

```suggestion


Refer to the [architecture](/docs/k8s/installation/install#architecture) section to learn more about the general architecture of Consul on Kubernetes.
For a hands-on experience with Consul as a service mesh
for Kubernetes, follow the [Getting Started with Consul service
mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.

You can install Consul on Kubernetes in two ways: 

1. [Helm chart install](#helm-chart-installation)
2. [Consul K8s CLI install <sup>ALPHA</sup>.](#consul-k8s-cli-installation)


```